routes/api.php

```diff
@@ -13,6 +13,7 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
+Route::middleware(['api'])->group(function (){
+   Route::get('/download','VideoDownloader@index');
```

Comment:
The route should be `Route::get('/download/{video_id}', 'VideoDownloader@getVideoDetails')`. do something like that. The video id is already parsed at the front end.